whisper_jax/modeling_flax_whisper.py

```diff
@@ -997,13 +997,16 @@ class FlaxWhisperPreTrainedModel(FlaxPreTrainedModel):
     def __init__(
         self,
         config: WhisperConfig,
-        input_shape: Tuple[int] = (1, 80, 3000),
+        input_shape: Tuple[int, int, int] = None,
         seed: int = 0,
         dtype: jnp.dtype = jnp.float32,
         params_dtype: jnp.dtype = jnp.float32,
         _do_init: bool = True,
         **kwargs,
     ):
+        if input_shape is None:
+            input_shape = (1, config.num_mel_bins, 2 * config.max_source_positions)
```

Comment:
Input shape taken from the model config -> this way it is compatible with the `FlaxWhisperPipline`, and also the `model` + `processor` API: https://github.com/sanchit-gandhi/whisper-jax#model-usage